go/vt/vttablet/tabletmanager/vreplication/stats.go

```diff
@@ -446,7 +477,7 @@ var vreplicationTemplate = `
     <th>State</th>
     <th>Stop Position</th>
     <th>Last Position</th>
-    <th>Seconds Behind Master</th>
+    <th>Seconds Behind Primary</th>
```

Comment:
Replication Lag?
I'm not going to hold up this PR, something to consider changing in the next one.